__init__.py

```diff
@@ -87,7 +87,7 @@ def poll(cls, context):
     def draw(self, context):
         col = self.layout.column()
         col.scale_y = 1.1  # extra padding
-        prop_split(col, context.scene, "f3d_type", "F3D Microcode")
+        prop_split(col, context.scene, "f3d_type", "Microcode")
```

Comment:
I find it weird to remove "F3D" from the display name there but keep it for the internal stuff, like the prop name ``f3d_type`` should be renamed to ``microcode_type`` or something similar

I guess it's more of a cleanup thing though so feel free to ignore this

Reply:
I'd disagree, every internal change to a proprety is just more upgrade code

Reply:
it would make more sense tbh, but yeah it's not the point of this PR so it can be done later